asset/js/setup.js

```diff
@@ -95,6 +95,21 @@ function setupSiteNav() {
   );
 }
 
+function setupReadingProgress() {
+  const h = document.documentElement;
+  const b = document.body;
+  const st = 'scrollTop';
```

Comment:
Is there a reason we are assigning these constant field names to a variable? Why not just do `document.body.scrollTop` etc. instead? It might be more verbose, but I think that is more readable compared to using ultra-short variable names.

Reply:
Resolved. :)